torch/_inductor/overrides.py

```diff
@@ -35,14 +35,22 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
             config.fallback_random
             and replacements[func] in replacements_using_triton_random
         ):
-            return replacements[func](*args, **kwargs)
+            if func not in fallback_cpu_random or (
+                func in fallback_cpu_random
```

Comment:
you don't have to check it again here? also, how about combining this if check with the one at line 34?